### PR TITLE
Feat/pro UI

### PR DESCRIPTION
**Pull request by BLKKKBVSIK**
In this PR:

- Basic UIs for the pro section of the DSC.
- CI Actions

Screenshot 2024-02-01 at 17 12 25
Screenshot 2024-02-01 at 17 12 33

### CODE REVIEW

File: packages/desktop_security_center/lib/src/widgets/feature_switch.dart [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                            ],
+                          ),
+                        ),
+                      ),
+                    ],
+                  ),
+                ],
+              ),
+            ),
+            Switch(
```
Comment (d-loose) [inline packages/desktop_security_center/lib/src/widgets/feature_switch.dart:77]:
```suggestion
            YaruSwitch(
```

**Comment by d-loose** [comment]
Looks very solid so far! :+1: UI code could use some trailing commas in a lot of places to increase legibility. But I suspect you plan to refactor it into smaller widgets anyways.

**Comment by BLKKKBVSIK** [comment]
If that's alright for you, I'll merge this bit, then come back to it after for the tests when starting to integrate the backend.
Will create a draft for the other section ASAP.

**Review by d-loose — APPROVED** [review]
LGTM :+1:

**Comment by Feichtmeier** [comment]
FYI there are also gnome-like switches available at 
https://github.com/ubuntu/yaru_widgets.dart
grafik
(nice app btw!)

**Pull request merged**